lib/i18n.js

```diff
@@ -99,11 +99,13 @@ export const i18n = {
             };
 
             render() {
-                let {children, ...params} = this.props;
-                return reactjs.createElement('span', {
+                const {children, tagType, props= {}, ...params} = this.props;
```

Comment:
To be honest, there's really low possibility that someone is using `tagType` or `props` key in ther translations, but right it's (unfortunately) breaking change.